src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from diffpy.labpdfproc.labpdfprocapp import WAVELENGTHS
+from diffpy.labpdfproc.tools import load_wavelength
+
+params1 = [
+    ([None, None], [0.71]),
+    ([None, "Ag"], [0.59]),
```

Comment:
maybe also test an unknown string?

Reply:
For an unknown string for anode_type, the function should still return default value, right?

Reply:
this is your decision.  What do you think the user would want?  Write the test for that.  Then implement it.

Reply:
I see code below but not the tests for these things....please make the tests first, then you can write the code.